tests/foreman/cli/test_remoteexecution.py

```diff
@@ -161,21 +161,24 @@ def test_positive_run_job_effective_user(self, rex_contenthost, module_target_sa
 
         :id: 0cd75cab-f699-47e6-94d3-4477d2a94bb7
 
-        :BZ: 1451675, 1804685
+        :BZ: 1451675, 1804685, 2258968
 
         :expectedresults: Verify the job was successfully run under the
             effective user identity on host
 
         :parametrized: yes
+
+        :customerscenario: true
         """
         client = rex_contenthost
         # create a user on client via remote job
         username = gen_string('alpha')
+        password = gen_string('cjk')
         filename = gen_string('alpha')
         make_user_job = module_target_sat.cli_factory.job_invocation(
             {
                 'job-template': 'Run Command - Script Default',
-                'inputs': f"command=useradd -m {username}",
+                'inputs': f"command=useradd -m {username} -p {password} -G wheel",
```

Comment:
Given running a job worked previously without the password specified (Using sudo?), are we positive that the password is really being used in this new version?

Reply:
I made some changes to reduce the possibility of secret sudo, but the only way I could find to be 100% sure is a negative assert -- running rex without pwd specified, so I added that as well